packages/suite/src/views/dashboard/components/AssetsView/components/AssetCoinLogo.tsx

```diff
@@ -29,9 +33,8 @@ export const AssetCoinLogo = ({ symbol, assetsFiatBalances, index }: CoinLogoPro
     return (
         <LogoWrapper>
             <Tooltip
-                content={`${(assetPercentage ?? 0).toFixed(2)} %`}
+                content={`${localizeNumber(assetPercentage ?? 0, locale, 2, 2)}%`}
```

Comment:
nit: if you have time would be sweet to have a util `localizePercentage(assetPercentage, locale)`

Reply:
or use Intl as we discussed – great idea!

Reply:
https://github.com/trezor/trezor-suite/pull/10973/commits/5cb05f59d9fe3ecdf48c945434b5424126ac4562